packages/mobile/src/account/LinkBankAccountScreen.tsx

```diff
@@ -0,0 +1,185 @@
+import VerificationComplete from 'src/icons/VerificationComplete'
+import VerificationDenied from 'src/icons/VerificationDenied'
+import VerificationPending from 'src/icons/VerificationPending'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import { KycStatus } from 'src/account/reducer'
+import { useTranslation } from 'react-i18next'
+import PersonaButton from 'src/account/Persona'
+interface Props {
+  kycStatus: KycStatus | undefined
+}
+
+function LinkBankAccountScreen({ kycStatus }: Props) {
+  return (
+    <SafeAreaView style={styles.body}>
+      <ScrollView
+        contentContainerStyle={styles.contentContainer}
+        showsVerticalScrollIndicator={false}
+      >
+        <StepOne kycStatus={kycStatus} />
+        <StepTwo disabled={!kycStatus || kycStatus !== KycStatus.Completed} />
+      </ScrollView>
+    </SafeAreaView>
+  )
+}
+
+function StepOne({ kycStatus }: Props) {
+  const { t } = useTranslation()
+
+  switch (kycStatus) {
+    case KycStatus.Created:
+      /* TODO: Ask Nitya on 12/1 about UI for Created Flow. I'll do it in separate PR */
+      return null
+    case KycStatus.Completed:
+      return (
+        <View style={styles.stepOne}>
+          <View style={styles.iconContainer}>
+            <VerificationComplete />
+          </View>
+          <Text style={styles.action}>{t('linkBankAccountScreen.completed.title')}</Text>
+          <Text style={styles.description}>{t('linkBankAccountScreen.completed.description')}</Text>
+        </View>
+      )
+    case KycStatus.Failed:
+      return (
+        <View style={styles.stepOne}>
+          <View style={styles.iconContainer}>
+            <VerificationDenied />
+          </View>
+          <Text style={styles.action}>{t('linkBankAccountScreen.failed.title')}</Text>
+          <Text style={styles.description}>{t('linkBankAccountScreen.failed.description')}</Text>
+          <View style={styles.button}>
+            <PersonaButton kycStatus={kycStatus} text={t('linkBankAccountScreen.tryAgain')} />
+          </View>
+          <TouchableOpacity
+            testID="SupportContactLink"
+            onPress={() => {
+              navigate(Screens.SupportContact)
+            }}
+          >
+            <Text style={styles.contactSupport}>{t('contactSupport')}</Text>
+          </TouchableOpacity>
+        </View>
+      )
+    case KycStatus.Pending:
+      return (
+        <View style={styles.stepOne}>
+          <View style={styles.iconContainer}>
+            <VerificationPending />
+          </View>
+          <Text style={styles.action}>{t('linkBankAccountScreen.pending.title')}</Text>
+          <Text style={styles.description}>{t('linkBankAccountScreen.pending.description')}</Text>
+          <Button
+            text={t('contactSupport')}
+            testID="SupportContactLink"
+            onPress={() => {
+              navigate(Screens.SupportContact)
```

Comment:
For a small improvement, I think it might be worth it to redirect to support with a prefilled text, example here https://github.com/valora-inc/wallet/blob/c1e0fae65108e064ce313f56ae85cbba84b1810f/packages/mobile/src/fiatExchanges/FiatExchangeAmount.tsx#L231